### DISCUSSION TITLE
Sending OpenTracing traces to Dynatrace via OpenTelemetry Protocol

### DISCUSSION THREAD

**Discussion by sasin07** [open-telemetry/community#1151] [Community Discussions]
Hi All,
Currently, I'm working with a legacy system that instrument the traces using OpenTracing. Is there any way to publish that generated traces to Dynatrace using OpenTelemetry protocol? 

Regards,
Sasin

**Comment by dmathieu** [comment]
Each language should be setting up an Opentracing shim, which will let you keep your opentracing calls and act as a proxy against the otel library.
Using that shim, you can start sending otel traces right now, and migrate away from opentracing later on.

**Comment by sasin07** [comment]
Hi Dmathieu,
Because we are working with a vendor product we cannot do any code-level changes. That product generates the traces using the Open Tracing protocol. With Opentracing shim approach is it possible to send the generated traces without any product-level code modifications?

**Reply by dmathieu** [reply]
Setting up the Opentracing shim will require code changes within your application, but not with the provider (dynatrace).
